src/ray/core_worker/store_provider/plasma_store_provider.h

```diff
@@ -95,6 +96,11 @@ class CoreWorkerPlasmaStoreProvider {
   Status Delete(const absl::flat_hash_set<ObjectID> &object_ids, bool local_only,
                 bool delete_creating_tasks);
 
+  /// Lists objects in used (pinned) by the current client.
+  ///
+  /// \return Output mapping of used object ids to (size, callsite).
+  absl::flat_hash_map<ObjectID, std::pair<int64_t, std::string>> UsedObjectsList();
```

Comment:
```suggestion
  absl::flat_hash_map<ObjectID, std::pair<int64_t, std::string>> UsedObjectsList() const;
```

Reply:
I don't think that's true, we do it [here](https://github.com/ray-project/ray/blob/master/src/ray/core_worker/reference_count.h#L175).

Reply:
Ah you probably just have to mark the mutex mutable.